helpers/config.py

```diff
@@ -274,14 +275,36 @@ def read_config(self):
         """
         config = {}
         try:
-            with open(Config.CONFIG_FILE, "r") as f:
+            base_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+            config_file = "{}/{}".format(base_dir, Config.CONFIG_FILE)
+            with open(config_file, "r") as f:
                 config = json.loads(f.read())
         except Exception as e:
```

Comment:
What exceptions should be caught here? `IOError` and `ValueError`?

Reply:
I guess IOError only!